src/lsdb/loaders/hipscat/abstract_catalog_loader.py

```diff
@@ -105,3 +93,12 @@ def _get_kwargs(self) -> dict:
         if self.config.dtype_backend is not None:
             kwargs["dtype_backend"] = self.config.dtype_backend
         return kwargs
+
+
+def read_pixel(
```

Comment:
Can we add a specific columns kwarg to this? The way `from_map` with dask expressions works is that if the method has a columns param then it use it for query optimization, but if not will just always load all columns.